graphql/directives.graphql

```diff
@@ -1,7 +1,11 @@
 directive @withSession on FIELD | FIELD_DEFINITION
 directive @withPermissions on FIELD | FIELD_DEFINITION
-directive @validateAdminUserAccess on FIELD | FIELD_DEFINITION
-directive @validateStoreUserAccess on FIELD | FIELD_DEFINITION
+directive @validateAdminUserAccess(
+  orgPermission: String
```

Comment:
I think we need to change `orgPermission` to something more meaningful... maybe `requiredPermission`?

Reply:
I agree